mlrun/serving/states.py

```diff
@@ -1218,12 +1243,90 @@ def __init__(
         self.raise_exception = raise_exception
         self.shape = "folder"
 
+    def add_shared_model_proxy(
+        self,
+        endpoint_name: str,
```

Comment:
```suggestion
        name: str,
```
I think that using `endpoint_name` is confusing, since it's basically the `name` of the model (or rather, of the proxy). For example, if I don't do MM at all then I won't really have a MEP so I might ask myself why I need to define an endpoint name?

I do understand that this is also the same in `add_model`, and I have the same question about that...

Reply:
We intentionally chose endpoint_name to clearly indicate that this parameter represents the model endpoint name. If I recall correctly, we discussed this in the HLD review and agreed on using this naming for clarity and consistency.

Also, just to clarify — even when model monitoring is disabled, the MEP (Model Endpoint) object will still be created. So the concept of an "endpoint name" still applies regardless of the monitoring state.

That said, if you feel strongly against this naming, we can reconsider — but personally, I think this name is more intuitive and aligns well with our existing conventions.

Reply:
@GiladShapira94 please advice what do you think?

Reply:
@theSaarco I spoke with @davesh0812, and I agree that we should indicate to the user that this parameter and the value the user provides affect the model_endpoint that has been created by deploying the model, in addition that creating a model endpoint for the deployed model.
Changing it to `name` will indicate something more global and less specific, and I think we should use the `endpoint_name`